cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/common/MeasureEvaluator.java

```diff
@@ -305,11 +305,20 @@ protected MeasureDef evaluate(
             String subjectTypePart = subjectInfo.getLeft();
             String subjectIdPart = subjectInfo.getRight();
             context.getState().setContextValue(subjectTypePart, subjectIdPart);
-
-            EvaluationResult result =
-                    libraryEngine.getEvaluationResult(id, subjectId, null, null, null, null, zonedDateTime, context);
-
-            evaluateSubject(measureDef, subjectTypePart, subjectIdPart, subjectSize, type, result);
+            try {
+                EvaluationResult result = libraryEngine.getEvaluationResult(
+                        id, subjectId, null, null, null, null, zonedDateTime, context);
+                evaluateSubject(measureDef, subjectTypePart, subjectIdPart, subjectSize, type, result);
+            } catch (Exception e) {
+                // Catch Exceptions from evaluation per subject, but allow rest of subjects to be processed (if
+                // applicable)
+                var error = String.format("Exception for subjectId: %s, Message: %s", subjectId, e.getMessage());
+                // Capture error for MeasureReportBuilder
+                if (!error.isEmpty()) {
```

Comment:
Why the conditional logic?  You created the error String so it'll never be empty